docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,36 @@
+
+# Envelope - Rule 5-43  
+
+**Rule ID:** 5-43  
+**Rule Description:** Automatic fenestration shading devices shall not be modeled in the Baseline.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** Section G3.1-5(d) Building Envelope Modeling Requirements for the Baseline building  
+**Mandatory Rule:** True
+
+**Manual Check:** None  
+**Evaluation Context:** Each SubSurface  
+**Function Call:**  
+
+NONE
+
+## Applicability Check:  
```

Comment:
There does not appear to be any applicability checks in the "Applicability Check" section. I don't understand why this loop is in this section

Reply:
from your comments, I think that you're suggesting that the rule should be evaluated at the building level, not the subsurface level. This makes sense. I updated the rule. & added an applicability check that the subsurfaces be exterior surfaces

Reply:
this whole section I think can be simplified to `if len(B_RMD...subsurfaces) == 0: NOT APPLICABLE`